ddl/ddl_api.go

```diff
@@ -1182,6 +1187,10 @@ func checkConstraintNames(constraints []*ast.Constraint) error {
 
 	// Set empty constraint names.
 	for _, constr := range constraints {
+		err := checkEmptyIndexInfo(constr.Name, constr.IsEmptyIndex)
```

Comment:
Is there a need to check it again? I think statements here are all preprocessed in planner.